uint64 timestamp		# time since system start (microseconds)

float32 roll_rate		# body frame roll rate setpoint (rad/s)
float32 pitch_rate		# body frame pitch rate setpoint (rad/s)
float32 yaw_rate		# body frame yaw rate setpoint (rad/s)